cmd/manager/main.go

```diff
@@ -175,6 +176,13 @@ func init() {
 }
 
 func execute() {
+	// update GOMAXPROCS to container cpu limit if necessary
+	_, err := maxprocs.Set(maxprocs.Logger(log.Info))
```

Comment:
I thought this library uses a package init function that runs as soon as you import the package
```
import _ "go.uber.org/automaxprocs"
```
Are we not setting it twice now (not sure if it matters much tbh)?

Reply:
Importing `go.uber.org/automaxprocs` triggers the `init`.  Importing `go.uber.org/automaxprocs/maxprocs` as Anya has done here should not trigger it (in theory).

Reply:
👍

Reply:
I should have explained this better in the PR. The [package docs](https://github.com/uber-go/automaxprocs/blob/master/maxprocs/maxprocs.go#L21) describe the use case a little bit better:
>// Package maxprocs lets Go programs easily configure runtime.GOMAXPROCS to
// match the configured Linux CPU quota. Unlike the top-level automaxprocs
// package, it lets the caller configure logging and handle errors.

So in our case it's just so we can inject our logger.